updates the alt attribute for aliasing (#19321)
mdn/content: files/en-us/web/media/formats/video_codecs/index.md

        </p>
      </td>
      <td>

>         <a href="moire-pattern.jpg"><img alt="a mat showing swirling effect casued due to the moire pattern" src="moire-pattern.jpg" /></a>

Reviewer: ```suggestion
        <a href="moire-pattern.jpg"><img alt="a brick wall showing swirling effect similar to waves due to the moire pattern" src="moire-pattern.jpg" /></a>
```

      </td>
      <td>
        <a href="staircase-effect.jpg"
          ><img alt="

>           an image showing diagonal lines appear to look like a staircase due to aliasing causing staircase effect

Reviewer: ```suggestion
          Photo of diagonal lines that look like a staircase due to aliasing causing a staircase effect
```
